### PR TITLE
[e2e fix] server: add nil checks to not panic

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca <runcom@redhat.com>

### CODE REVIEW

File: server/container_exec.go [hunk 1]
```diff
@@ -29,8 +29,11 @@ func (s *Server) Exec(ctx context.Context, req *pb.ExecRequest) (*pb.ExecRespons
 
 // Exec endpoint for streaming.Runtime
 func (ss streamService) Exec(containerID string, cmd []string, stdin io.Reader, stdout, stderr io.WriteCloser, tty bool, resize <-chan term.Size) error {
-	fmt.Println(containerID, cmd, stdin, stdout, stderr, tty, resize)
-	c := ss.runtimeServer.state.containers.Get(containerID)
+	c := ss.runtimeServer.GetContainer(containerID)
+
+	if c == nil {
+		return fmt.Errorf("could not find container %q", containerID)
+	}
```
Comment (sameo) [inline server/container_exec.go]: Would something like:
```
if c := ss.runtimeServer.GetContainer(containerID); c == nil {
      return fmt.Errorf("could not find container %q", containerID)
}
```

look more readable  ?

Comment (runcom) [inline server/container_exec.go]: that's not going to work as `c` lives only during the `if` lifetime :( and can't be used below.

Comment (sameo) [inline server/container_exec.go]: you're right, sorry for the noise.

**Comment by sameo** [comment]
LGTM

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**